community/cowsay/sources

```diff
@@ -1,2 +1 @@
-https://github.com/tnalpgge/rank-amateur-cowsay/archive/cowsay-3.04.tar.gz
-patches/cowsay.patch
+git+https://github.com/cowsay-org/cowsay.git@vVERSION
```

Comment:
why make kiss clone the repo, then checkout the tag, rather than using the release tarball?

https://github.com/cowsay-org/cowsay/archive/refs/tags/v3.7.0.tar.gz